src/goto-symex/postcondition.cpp

```diff
@@ -155,7 +158,8 @@ bool postconditiont::is_used(
   if(expr.id()==ID_address_of)
   {
     // only do index!
-    assert(expr.operands().size()==1);
+    DATA_INVARIANT(
+      expr.operands().size() == 1, "address_of_exprt takes one operand.");
```

Comment:
See above: use `to_address_of_expr` etc - I'll stop putting the same comments, it applies several times below.